src/button/icon-button.stories.ts

```diff
@@ -134,3 +134,33 @@ WithAutoAlign.args = {
 	align: "top",
 	isOpen: true
 };
+
+
+const DisabledWithTooltipTemplate = (args) => ({
+	props: args,
+	template: `
+		<cds-icon-button
+			buttonId="icon-btn1"
+			type="button"
+			[kind]="kind"
+			[size]="size"
+			[align]="align"
+			[autoAlign]="autoAlign"
+			[buttonNgClass]="buttonNgClass"
+			[disabled]="true"
+			[description]="description"
+			(click)="onClick($event)"
+			(mouseenter)="onMouseEnter($event)"
+			(mouseleave)="onMouseLeave($event)"
+			(focus)="onFocus($event)"
+			(blur)="onBlur($event)">
+			<svg class="cds--btn__icon" cdsIcon="copy" size="16"></svg>
+		</cds-icon-button>
+	`
+});
+export const DisabledWithTooltip = DisabledWithTooltipTemplate.bind({});
+DisabledWithTooltip.args = {
+	autoAlign: true,
+	description: "Icon Description",
+	align: "top"
+};
```

Comment:
Probably don't need a separate story to showcase this

Reply:
Was for me to make testing easier, but will remove :)